### ISSUE TITLE
Non-digital member Gifiting

### ISSUE THREAD

**Issue by JonnyBuck16** [yalla-coop/tempo#128]
Hi all, 

Occurred to me last night that non-digital members are unable to gift their Time Credits or receive gifted Time Credits as part of the member journeys we have at present.

Think that we could help members receive gifted Time Credits from digital members by allowing digital members to input a membership ID on the gifting journey, rather than sending out a link #51 

Not sure how we can enable non-digital members to give their Time Credits to others (either digital or non-digital) without going via their group, and this would need new flows for Cutomers...

Any thoughts guys?

**Comment by harrietepugh** [comment]
@JonnyBuck16 members gifting to non-digital members using a member ID is a great idea and we should do this. I think the greater need from a non-digital member POV is to be able to gift... in many cases they'll be gifting to digital members (who aren't signed up)... I think we need to write up a non-digital support pack for Customers. A series of cards or a one-pager on what you can do for non-digital members

**Comment by harrietepugh** [comment]
@JonnyBuck16 one issue with a group lead being able to gift on a non-digital members' behalf is that its open to abuse. What if the group lead gifts the NDM's TCs back to the community group? Argh

**Comment by harrietepugh** [comment]
Maybe this is just something they have to sign up properly for....

**Comment by harrietepugh** [comment]
Suggested content for group leads around supporting non-digital members:

SUPPORTING A MEMBER WITHOUT DIGITAL ACCESS

1.	Register your member
	Enter your member’s full name and DOB to open a member account and generate a Membership ID
	Fill out a membership card with their membership ID and issue it to your member
2.	Notify members that they have Time Credits in their account
	When distributing Time Credits to your non-digital members, make sure they know they have Time Credits to spend
3.	Provide information on where you member can spend Time Credits
	To spend Time Credits, members need to show their membership ID to staff in the venues where they want to spend Time Credits
	Make sure your members know where and when they can spend Time Credits and any restrictions on those activities (e.g. weekdays only)
4.	Check if your member has Time Credits in their account
	You can view how many Time Credits you have given to a member by viewing their details in your volunteer records
	To check their balance, your member should contact Tempo on…
5.	Enable your member to Give Time Credits 
	To Give Time Credits back to their community groups or to friends and family, your member will need to sign up for a digital account. This is to ensure Time Credits transactions are secure



@JudeLuckett @JonnyBuck16 @thejoefriel I think we will have to make non-digital members open digital accounts in order to Give Time Credits. If Group Leads can do it, it opens up a whole can of worms around people controlling others' TC transactions. We can test this at least.... thoughts?

P.s. Jonny I know I've referred to members as non-digital here but I think its OK given the context. All the rules make it quite obvious what it means :)

**Comment by JudeLuckett** [comment]
I agree on gifting. People can still take others with them but I think this is OK at least for now

**Comment by thejoefriel** [comment]
I also agree on this @harrietepugh . I think there has to be some limits on functionality if they don't have a full digital account. 

I also think this run through explanation makes complete sense too 👍
